ha/monitor/k8s/parser.py

```diff
@@ -37,7 +37,7 @@ def parse(self, an_event, cached_state):
 class NodeEventParser(ObjectParser):
     def __init__(self):
         super().__init__()
-        self._type = 'node'
+        self._type = 'host'
```

Comment:
As pod is changed to node. I have changed node to host. I can change this name. Suggestions are open

Reply:
This Change is fine

Reply:
As decided, No change here